core/client/kt/requests.go

```diff
@@ -75,7 +77,7 @@ func (v *Verifier) CreateUpdateEntryRequest(
 	if err != nil {
 		return nil, err
 	}
-	previous := objecthash.ObjectHash(getResp.GetLeafProof().GetLeaf().GetLeafValue())
+	previous := objecthash.ObjectHash(prevEntry)
```

Comment:
Thanks, I've added a TODO. 
I'm not sure what the second part of the comment means: `prevEntry` is of type  `Entry` and not []byte. Should I add a comment about this as well?